JDBC/OAuthSampleApp/src/main/java/OAuthSampleApp.java

```diff
@@ -211,41 +204,57 @@ private static void GetAndSetTokens(Map<String, String> formData) throws Excepti
 	    	    String res = buf.toString("UTF-8");
 	        	JsonElement jElement = new JsonParser().parse(res);
 	            JsonObject jObject = jElement.getAsJsonObject();
-	            
 				// set Tokens as System Properties - new values to access_token and refresh_token
-				System.setProperty(OAUTH_ACCESS_TOKEN_VAR_STRING, jObject.get("access_token").getAsString());
-				System.setProperty(OAUTH_REFRESH_TOKEN_VAR_STRING, jObject.get("refresh_token").getAsString());
-
+	            if(null == jObject.get("access_token").getAsString().isEmpty() || jObject.get("access_token").getAsString().isEmpty())
+	            {
+	            	throw new Exception("Access Token is empty, Please verify the config.properties for proper inputs");
+	            }else 
+	            {
+					System.setProperty(OAUTH_ACCESS_TOKEN_VAR_STRING, jObject.get("access_token").getAsString());
+	            }
+	            if(null == jObject.get("refresh_token").getAsString().isEmpty() || jObject.get("refresh_token").getAsString().isEmpty())
+				{	
+					throw new Exception("Refresh Token is empty, Please verify the config.properties for proper inputs");
+	            }else
+				{
+					System.setProperty(OAUTH_REFRESH_TOKEN_VAR_STRING, jObject.get("refresh_token").getAsString());
+				}
 	        } catch (Exception e) {
 	            String res = "";
-	            try {
+	            try 
+	            {
 	                BufferedInputStream in = new BufferedInputStream(connection.getErrorStream());
 	                ByteArrayOutputStream buf = new ByteArrayOutputStream();
-	                for (int result = in.read(); result != -1; result = in.read()) {
+	                for (int result = in.read(); result != -1; result = in.read()) 
+	                {
 	                    buf.write((byte) result);
 	                }
 	                res = buf.toString("UTF-8");
-	            } catch (Exception ex) {
+	            } catch (Exception ex) 
+	            {
 	                //Improper setup. Passes in SF, fails in Devjail. Skip when this happens, but print the error.
 	                ex.printStackTrace();
 	                System.out.println("Error when fetching token endpoints. Skipping test.");
```

Comment:
Remove this message or make it more clear.

Reply:
The message is removed, And it will print the printStackTrace if any exceptions.